components/StartInput.js

```diff
@@ -34,36 +45,48 @@ function StartInput({ setDataToStore, state }) {
     setData({ ...data, ...{ lunghezza: Number(e.target.value) } });
```

Comment:
utilizza sempre o l'italiano o l'inglese. 

Visto che il framework e' in inglese, usa l'inglese